### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by anq420**
image

### CODE REVIEW

File: lesson_19_hw/lesson_19_hw.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+app = Flask(__name__)
+app.secret_key = 'QwErTyUiOp[123456789010]'
+
+metadata = MetaData()
+engine = create_engine('postgresql+psycopg2://anq420:030499@localhost/lesson_19_hw_contract_first')
+ses = Session(bind=engine)
+
+
+users = Table(
```
Comment (nicilytop-tms) [inline lesson_19_hw/lesson_19_hw.py]: Better to separate it on different modules

File: lesson_19_hw/lesson_19_hw.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
+from flask import (Flask, render_template, request, flash)
+from sqlalchemy import (create_engine, MetaData, Table, Integer, String, Column, CheckConstraint)
+from sqlalchemy.orm import Session
+
+
+app = Flask(__name__)
+app.secret_key = 'QwErTyUiOp[123456789010]'
+
+metadata = MetaData()
```
Comment (nicilytop-tms) [inline lesson_19_hw/lesson_19_hw.py]: does it work?

File: lesson_19_hw/lesson_19_hw.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+        check_nickname = [i[3] for i in ses.query(users).all()]
+
+        email = request.form['email']
+        password = request.form['password']
+        password_2 = request.form['password_2']
+        nickname = request.form['nickname']
+
+        if password == password_2 and email not in check_email and nickname not in check_nickname:
+            try:
+                new_account = users.insert().values(email=email, password=password, nickname=nickname)
```
Comment (nicilytop-tms) [inline lesson_19_hw/lesson_19_hw.py]: we should hash password

**Review by nicilytop-tms — CHANGES_REQUESTED** [review]
Hash your password please

File: lesson_19_hw/lesson_19_hw.py (hunk 2 above)
Comment (anq420) [reply lesson_19_hw/lesson_19_hw.py]: What do you mean?

**Review by nicilytop-tms — APPROVED** [review]

**Pull request merged**